code/modules/reqs/supplypacks.dm

```diff
@@ -507,6 +507,29 @@ EXPLOSIVES
 /datum/supply_packs/explosives/detpack
 	name = "detpack explosives"
 	contains = list(
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/assembly/signaler,
+	)
+	cost = 10
+
+/datum/supply_packs/explosives/detpack
```

Comment:
this is a duplicate definition of the one above it.

Reply:
Intended. It's to have multiple detpack. Req code is wack.